lang/src/OTS/ITF/org/partiql/ots/operator/ScalarOpId.kt

```diff
@@ -0,0 +1,14 @@
+package OTS.ITF.org.partiql.ots.operator
+
+enum class ScalarOpId {
+    NOT,
+    POS,
+    NEG,
+    PLUS,
+    MINUS,
+    TIMES,
+    DIVIDE,
+    MODULO,
+    CONCAT,
+    LIKE
+}
```

Comment:
@lziq , this is probably an omission after we discussed this. Please take this out. It is not being used anywhere.  To experiment with this idea, please use a separate branch/PR.

Reply:
No rush.  It is ok to do as part of follow-up work in either PR816 or 818.

Reply:
Removed it here, https://github.com/partiql/partiql-lang-kotlin/pull/818/files